easybuild/easyconfigs/e/Elk/Elk-4.0.15-intel-2016.01.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = 'MakeCp'
+
+name = 'Elk'
+version = '4.0.15'
+
+homepage = 'http://elk.sourceforge.net/'
+description = """An all-electron full-potential linearised augmented-plane wave (FP-LAPW) code with
+many advanced features. Written originally at Karl-Franzens-Universität Graz as a milestone of the
+EXCITING EU Research and Training Network, the code is designed to be as simple as possible so that
+new developments in the field of density functional theory (DFT) can be added quickly and reliably.
+"""
+
+toolchain = {'name': 'intel', 'version': '2016.01'}
+toolchainopts = {'usempi': True}
+
+sources = [SOURCELOWER_TGZ]
+source_urls = [SOURCEFORGE_SOURCE]
+
+checksums = [
+    '760623faf5aab9b1f49488edbaae65fb',  # elk-4.0.15.tgz
+]
+
+dependencies = [
+    ('libxc', '2.2.3'),
+]
+
+
+# make.inc file is generated interactively by "setup" command, creating it here
+prebuildopts = 'echo "F90_OPTS = $FFLAGS" > make.inc && '
+prebuildopts += 'echo "F77_OPTS = $FFLAGS" >> make.inc && '
+prebuildopts += 'echo "LIB_LPK = $LIBLAPACK" >> make.inc && '
```

Comment:
@migueldiascosta let me know if you're happy with this as-is; with the successful test reports this is good to merge if you ask me